desktop-widgets/diveplanner.cpp

```diff
@@ -78,6 +78,9 @@ DivePlannerWidget::DivePlannerWidget(QWidget *parent) : QWidget(parent, QFlag(0)
 	QShortcut *closeKey = new QShortcut(QKeySequence(Qt::Key_Escape), this);
 	connect(closeKey, &QShortcut::activated, plannerModel, &DivePlannerPointsModel::cancelPlan);
 
+	QShortcut *mirrorProfileShortcut = new QShortcut(QKeySequence(Qt::Key_M + Qt::CTRL), this);
```

Comment:
Making this available only as a shortcut creates the problem that there will be no way for users to discover that this functionality exists, and consequently they will not be using it.

The fact that you have added a mention to the user manual is a good start, but probably does little to address the issue:
- not many users read 5000 lines of user manual for every release to discover new features hidden behind shortcuts;
- the user manual only exists in English - effectively we are shutting out any non English speakers from ever using this (and to make matters worse, for them `[Ctrl] - M` most likely isn't intuitive or easy to remember).

In order to create a User Experience that allows users to discover and use this, I think at least one of the following should be considered:
- adding a button for it (obviously clutters the UI, but provides a better UX);
- adding an entry for it to the menubar (a bit problematic as at the moment the menubar is not updated when the planner is opened, and therefore doesn't have much to provide to users while in the planner;
- adding a tooltip that is showing when the keyboard shortcut can be used (not great UX, but better than not allowing users to discover that the functionality exists.

In addition to this, an entry should be added to the [changelog](https://github.com/subsurface/subsurface/blob/master/CHANGELOG.md), giving the user the information required to find this shortcut (again only a limited mitigation, as not many users read the changelog, and even fewer remember it when they get to the point of wanting to use the functionality).